itest/src/custom-rule/build.sc

```diff
@@ -5,7 +5,7 @@ import mill.scalalib._
 import os._
 
 object project extends ScalaModule with ScalafixModule {
-  def scalaVersion           = "2.13.2"
+  def scalaVersion           = "2.13.3"
```

Comment:
I'm super new to Mill so I'm unsure if this is even possible or not, but since you're extending `BuildInfo` in your main build.sc, do you have access to that here so you can just pull the `scalaVersion` from there so you don't have to also hardcode it here?

Reply:
This is integration test, so we might want to test a project for every Scala version. Not sure if it make sense to not hard code the version.